modules/secure_storage_connector/main.tf

```
@@ -0,0 +1,55 @@
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {
+}
+
+data "azurerm_resource_group" "group" {
+  name = var.resource_group_name
+}
+
+resource "azurerm_user_assigned_identity" "default" {
+  name                = "${var.namespace}-identity"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+}
+
+resource "azurerm_federated_identity_credential" "app" {
+  parent_id           = azurerm_user_assigned_identity.default.id
+  name                = "${var.namespace}-federated-credential"
```

```suggestion
  name                = "${var.namespace}-federated-credential"
```

Do you know the length limit on this?

---

120 characters https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust-user-assigned-managed-identity?pivots=identity-wif-mi-methods-azp#important-considerations-and-restrictions